test/mycnf/full-dsn

```diff
@@ -3,6 +3,5 @@ user     = U
 password = P
 host     = H
 port     = 33560
-ssl-ca   = CA
-ssl-cert = CERT
-ssl-key  = KEY
+# Path is relative to the test file factory_test.go
+ssl-ca   = ../test/mycnf/rds-master.pem
```

Comment:
Would be nice to get this to just `./rds-master.pem` but currently the path should be relative to the script that's using this full-dsn file.